hq_superset/models.py

```diff
@@ -32,3 +45,71 @@ def update_dataset(self):
                 rows = list(cast_data_for_table(self.data, table))
                 insert_stmt = table.insert().values(rows)
                 connection.execute(insert_stmt)
+
+
+class OAuth2Client(db.Model, OAuth2ClientMixin):
+    __bind_key__ = OAUTH2_DATABASE_NAME
+    __tablename__ = 'hq_oauth_client'
```

Comment:
nit: building on the conversation about naming, this could be `commcarehq_oauth_client` or `commcare_hq_oauth_client`